tensorflow/lite/examples/python/label_image.py

```diff
@@ -93,3 +99,5 @@ def load_labels(filename):
       print('{:08.6f}: {}'.format(float(results[i]), labels[i]))
     else:
       print('{:08.6f}: {}'.format(float(results[i] / 255.0), labels[i]))
+
+  print("time: ", stop_time - start_time)
```

Comment:
nit: wrap string with single quote
remove extra space after ":", the print statement adds space between arguments.

Preferably would be nicer to format this as something like `'time: {:.3f}ms'.format((stop_time - start_time) * 1000)`